x/pylons/client/cli/tx_execute_recipe_test.go

```diff
@@ -675,32 +675,32 @@ func TestExecuteRecipeItemInputOutput(t *testing.T) {
 			ID: "itemInputID",
 			Doubles: []types.DoubleInputParam{
 				{
-					Key:  "main",
+					Key:      "main",
 					MinValue: sdk.NewDec(50),
 					MaxValue: sdk.NewDec(100),
 				},
 			},
 			Longs: nil,
 			Strings: []types.StringInputParam{
 				{
-					Key:     "testInput",
-					Value:   "testVal",
+					Key:   "testInput",
+					Value: "testVal",
 				},
 			},
 		},
 	})
 	require.NoError(t, err)
 
-	//itemInputIDs, err := json.Marshal([]string{types.EncodeItemID(0)})
+	// itemInputIDs, err := json.Marshal([]string{types.EncodeItemID(0)})
```

Comment:
same here, is it dead code or necessary for testing?

Reply:
Same thing